### DISCUSSION TITLE
Side navigation bar

### DISCUSSION THREAD

**Discussion by Hezel2000** [quarto-dev/quarto-cli#4214] [Q&A] [answered]
I am trying to get a side navigation bar to my quarto website, but it is not working. I tried numerous variations in the _publish.yml entry, save it, and publish it to quarto-pub via VS Code. I see the website, it updates when I change something in contents – but no side navigation bar. Currently, my _publish.yml file looks as follows – but as said, I tried many variations, starting with copy&paste the example from the Quarto Docs (I think the indentations render incorrectly here):

- source: Tutorials & Documentation.qmd
  quarto-pub:
    - id: aee9fd50-e9ab-4684-83b1-df5c0ec4114b
      url: 'https://quartopub.com/sites/hezel2000/flank-method-data-reduction-workflow-tutorials--documentation'
  project:
    type: website
  website:
    navbar:
      background: primary
      search: true
      left:
        - text: Home
          file: index.qmd
        - contents1.qmd
        - contents2.qmd

**Comment by mcanouil** [comment, marked as answer]
Can you provide a repository with your project?
Or use code blocks here with the content of `_quarto.yml` (as in Quarto/RMarkdown)?

By the way, `_publish.yml` is not the file to define the structure or where to add any configuration related to your Quarto project/document, see <https://quarto.org/docs/websites/website-navigation.html#side-navigation>.
See <https://quarto.org/docs/publishing/quarto-pub.html#publish-command> for what `_publish.yml` is about.

(Note you need to use four `` ` `` to include content with code cells.)

**Comment by Hezel2000** [comment]
You are right, I confused _quarto.yml and _publish.yml. I just dived into this a little bit. I started from my Python project, as I wanted to include code, but it should primarily be a website. I realise that I cannot simply add a _quarto.yml to what I have at the moment. Likely there are differences between websites, sites with code, etc. I have to check this out further.

The repo would be: https://github.com/Hezel2000/flank_method

I mark this answered for the moment and until I informed myself about these differences further. Thanks!

**Comment by Hezel2000** [comment]
I got it! This is all working now – thanks again, great! I wasn't aware of these differences.
